CHANGES

```diff
@@ -1,6 +1,7 @@
 243
 
 - Use pipe as separator for openmetrics object names config
+- Update jackson to 2.17.0
```

Comment:
This also fixes a bug (from  ObjectMapperProvider's user perspective). Worth calling it out in CHANGES?

Reply:
this comes with the version upgrade, isn't it? There are other bugs solved between 2.16.1 and 2.17.0 (3 releases in between)